.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 🐞 Bug report
+about: Report a bug in the Ember Framework. Before you create a new issue, please
+  search for similar issues. It's possible somebody has encountered this bug already.
+title: "[Bug] Bug report"
+labels: ''
+assignees: ''
+
+---
+
+### 🐞 Describe the Bug
+A clear and concise description of what the bug is.
+
+### 🔬 Minimal Reproduction
+Describe steps to reproduce. If possible, please, share a link with a minimal reproduction.
+
+<!--
+Create a minimal reproduction using one of the following:
+- Ember Twiddle: http://ember-twiddle.com/
+- Create a GitHub repository: https://guides.emberjs.com/release/getting-started/quick-start/
+-->
+
+### 😕 Actual Behavior
+A clear and concise description of what is happening.
+
+### 🤔 Expected Behavior
+A clear and concise description of what you expected to happen.
+
+### 🌍 Environment
+
+- Ember: 3.18
```

Comment:
Let's not include the Ember version in the template (I'll forget to update).

Reply:
That's a fair point. Done!